semgrep-core/bin/Main.ml

```diff
@@ -8,6 +8,7 @@
  *    May you share freely, never taking more than you give.
  *)
 open Common
+open Version
```

Comment:
I try to avoid open. I just do it for Common and when you really need to reference lots of entities
from another file (e.g., I often do open Ast_xxx), but here I think it's better to qualify at use time and
do Version.version

Reply:
I didn't know ocaml auto resolved Version.version and thought open was like python import :D